Assignment1/test-output/Default suite/Default test.html

```diff
@@ -57,9 +57,9 @@ <h2 align='center'>Default test</h2><table border='1' align="center">
 <tr>
 <td>Tests passed/Failed/Skipped:</td><td>7/0/0</td>
 </tr><tr>
-<td>Started on:</td><td>Fri Jul 17 16:38:37 BST 2020</td>
+<td>Started on:</td><td>Wed Jul 22 12:35:59 BST 2020</td>
```

Comment:
Test output should not be a part of the repository.

Reply:
removed test-output folder from repository